src/AsyncAPI.Saunter.Generator.Cli/Commands/Tofile.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
we have nothing to do with .NET Foundation.

it is better to delete these comments, all licensing information is provided in a separate file

Reply:
My mistake, it was probably part of a template, while I thought it was part of the Saunter .editorconfig defaults.

I will look into it, as well as your other comments. Thanks for reviewing it so quickly. 
About the tests: yes they are actually e2e tests, I didn't know how to test it differently.

I uploaded a new nuget package with -beta suffix that I'm already using it for work related stuff. (same with the .Build nuget package)

Reply:
This WAS actually in the .editorconfig "file_header_template", I've removed it as it wasn't desired.